pages/docs/tutorials/getting-started/hello-world-receive-message.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Receive 'Hello world' message"
```

Comment:
the only difference in this document is receive operation and there is a lot of repetition - is it intended? especially that app name is the same `Hello world application`. Shouldn't it be done like other tuturials for example for Kafka, where first tutorial would be about send message, and the one about receiving would be "extending" the send-one, where you extend the document with additional receive operation (of course still explaining these can be separate)